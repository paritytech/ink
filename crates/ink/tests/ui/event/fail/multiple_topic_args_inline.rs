#[ink::event]
pub struct Event {
    #[ink(topic, topic)]
    pub topic: [u8; 32],
}

fn main() {}