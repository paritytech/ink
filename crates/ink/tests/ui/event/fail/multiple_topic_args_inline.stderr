error: Invalid `#[ink(topic)]` attribute: multiple arguments not allowed.
 --> tests/ui/event/fail/multiple_topic_args_inline.rs:3:11
  |
3 |     #[ink(topic, topic)]
  |           ^^^^^
